zerver/migrations/0271_migrate_alert_words.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from django.db import migrations
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+def move_to_seperate_table(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    UserProfile = apps.get_model('zerver', 'UserProfile')
+    AlertWords = apps.get_model('zerver', 'AlertWords')
+
+    for user_profile in UserProfile.objects.all():
+
+        list_of_words = user_profile.alert_words
+        if list_of_words != "[]":
```

Comment:
Please use `ujson.loads()` rather than manually parsing JSON here; there's no reason to do it this way.